ecs-init/docker/docker.go

```diff
@@ -440,6 +463,32 @@ func (c *client) getHostConfig(envVarsFromFiles map[string]string) *godocker.Hos
 	return createHostConfig(binds)
 }
 
+// getCredentialsFetcherSocketBind returns the bind for CredentialsFetcher socket and also whether the bind exists.
+// Value for the bind is as follow:
+// CREDENTIALS_FETCHER_HOST (as in os.Getenv) set: source CREDENTIALS_FETCHER_HOST (as in os.Getenv, trim unix:// prefix),
+// dest CREDENTIALS_FETCHER_HOST (as in /etc/ecs/ecs.config, trim unix:// prefix)
+func getCredentialsFetcherSocketBind(envVarsFromFiles map[string]string) (string, bool) {
+	CredentialsFetcherEndpointAgent := defaultCredentialsFetcherSocketPath
+	CredentialsFetcherUnixSocketSourcePath, fromEnv := config.CredentialsFetcherUnixSocket()
+	if fromEnv {
+		if credentialsFetcherEndpointConfig, ok := envVarsFromFiles[config.CredentialsFetcherHostEnvVar]; ok && strings.HasPrefix(credentialsFetcherEndpointConfig, config.UnixSocketPrefix) {
```

Comment:
nit: any way to make this line more readable?  There's a lot going on here.

Reply:
made the change